### PR TITLE
docs: Update sentence for clarity (Arijus' input)

### PR DESCRIPTION
**Pull request by pradeephubgit**
Thanks to @argshook for suggestion. 

**Note** - Pls ignore 'blog/2022-01-27-release-sql-jit-compiler.md' file. It is displayed by mistake.

### CODE REVIEW

File: blog/2022-01-27-release-sql-jit-compiler.md [hunk 1]
```diff
@@ -208,7 +208,7 @@ working on replication, further JIT-compiled filter performance improvements,
 and more.
 
 We hope you enjoyed the features and functionality in version 6.2. See the
-[release notes on GitHub](https://github.com/questdb/questdb/releases/tag/6.2.0)
+[release notes on GitHub](https://github.com/questdb/questdb/releases/tag/6.2)
```
Comment (argshook) [inline blog/2022-01-27-release-sql-jit-compiler.md:211]: Hey @pradeephubgit thank you for submitting the PR!

If this file is not meant to be merged, then please revert this change. There's no ability to merge only part of the PR.

Comment (pradeephubgit) [reply blog/2022-01-27-release-sql-jit-compiler.md:211]: Hi @argshook - I am really struggling to remove this file from appearing in every new PR. It is automatically appearing, though I have not made any updates. Sure, i will try reverting the change and check. Thanks for the suggestion.

Comment (pradeephubgit) [reply blog/2022-01-27-release-sql-jit-compiler.md:211]: I don't see the [Revert](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/reverting-a-pull-request) option. I will check if someone has to enable this option for me. Thanks again.

Comment (argshook) [reply blog/2022-01-27-release-sql-jit-compiler.md:211]: I'm not sure but my guess is that you maybe create edits from a branch which is other than `master`?

I assume the way you create PRs is by clicking the `Edit` button on github. If that's the case, please double check if that `Edit` button is clicked on a page, which points to `master` branch. You can see that at the top left side of every page on github.

image

Comment (pradeephubgit) [reply blog/2022-01-27-release-sql-jit-compiler.md:211]: No, I fork from the main repo (in QuestDB), navigate to the respective file in my local repo, update it, create a PR in my local repo, and then go back to the QuestDB repo, and create a new PR there by comparing across forks. 
I guess this started after creating a few PRs.

Comment (pradeephubgit) [reply blog/2022-01-27-release-sql-jit-compiler.md:211]: I understood your earlier point about Edit now, Arijus. I use Edit to update content. I will check one again and revert.

Comment (pradeephubgit) [reply blog/2022-01-27-release-sql-jit-compiler.md:211]: image

Comment (pradeephubgit) [reply blog/2022-01-27-release-sql-jit-compiler.md:211]: When I click Edit, it points to 'master'.

Comment (argshook) [reply blog/2022-01-27-release-sql-jit-compiler.md:211]: I usually create and push commits changes through command line, so i'm not an expert on github UI.

But, i believe the root cause of this these seemingly random changes is
due to the fork being out of sync.

Check here, I think this article should help understand the issue
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork

---

If the issue persists, I can submit your changes on my own, in a separate PR. But in that case the changes would point to my account and i don't want to take away your contribution and make it look like it was mine :)

Comment (pradeephubgit) [reply blog/2022-01-27-release-sql-jit-compiler.md:211]: This is indeed your suggestion, Arijus sir. Since the intention is to make customers' experience easier, it doesn't matter who submits it. I will be happy if you can submit the changes.

Comment (pradeephubgit) [reply blog/2022-01-27-release-sql-jit-compiler.md:211]: Sure, Arijus. I will go through the article you sent, and then take action. I cleaned up the Overview of the open PRs, as per the [link](https://www.conventionalcommits.org/en/v1.0.0/) from Pei-Shun. I am not sure if it is good enough, yet. 
Now, I will go through the info that you sent. 

Thanks for patiently giving me tips. I will incorporate them (for existing PRs) and request you take a look, if you are okay with it. After you are satisfied with the changes, I will create new PRs. 

I am not discouraged by your candid advice. Thanks again for your guidance.

Regards,
Pradeep.

Comment (pradeephubgit) [reply blog/2022-01-27-release-sql-jit-compiler.md:211]: Hi @Pei @argshook, @pswu11  - Since yesterday, I am trying to resolve my PR issues. My progress so far:

1. In the 'Files changed' tab, barring one PR, only needed files appear in all other PRs now. 'Fetch upstream' is clean for these branches/PRs. 
2. I created a new PR today, wherein additional files don't appear, and 'Fetch upstream' shows no conflicts too. 

Two issues persist:
1. In the 'Commit' tab, an unneeded commit/file appears. 
2. One PR still has 'Fetch upstream' conflicts. 

I will try to resolve these issues ASAP, so that it is easy for you (and other reviewers) to review. 

Thanks again for the guidance,
Pradeep.

Comment (argshook) [reply blog/2022-01-27-release-sql-jit-compiler.md:211]: > Maybe this is a dumb point, but if we push my local deletion state to the questdb.io repo, will it solve the problem?

In the end what matters is what changes end up in a PR. Thus, while you're working on your fork, you are free to do any changes you like, push commits, experiment to see what happens if you're not sure.

Even after you submit a PR to questdb.io and notice something that shouldn't be in that PR, you can still push more stuff and change things.

I see you managed to clear the PRs from unrelated changes, great work!

Comment (pradeephubgit) [reply blog/2022-01-27-release-sql-jit-compiler.md:211]: Thanks a lot, @argshook. I was embarrassed since my work had inconvenienced others (you and Pei). I learned a few things about GitHub in the last few days.

Comment (argshook) [reply blog/2022-01-27-release-sql-jit-compiler.md:211]: don't worry about it, we're all learning things as we go. Thanks for your contributions!
